File: responses/YXVDK8-JB4/count_of_largest_integer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python

    A.sort()

    counter = 0

    for i in range(0, len(A)):
        if A[i] == A[i-1]:
            counter += 1

    return counter

File: responses/RVM3S9-UP8/count_of_largest_integer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript
    var largest_N = A[0];
    var counter = 1;
    for (i=1; i<A.length; i++){
        if (A[i]==largest_N){
            counter++;
        } else if (A[i]>largest_N){
            largest_N = A[i];
            counter = 1;
        }
    }
    return counter
}
